### PR TITLE
Fleet mgmt

### PR DESCRIPTION
**Pull request by aedsinger**
Check out the changes to stretch_install_user to support stretch_update.sh

### CODE REVIEW

File: stretch_install_user.sh [hunk 1]
```diff
@@ -1,15 +1,27 @@
...
+if [ "$(ls -A ~/stretch_user)" ]; then
+    echo "###########################################"
+    echo "UPDATING USER SOFTWARE"
+    UPDATING=true
+else
+    UPDATING=false
+    echo "###########################################"
+    echo "NEW INSTALLATION OF USER SOFTWARE"
+    echo "UPDATE .bashrc FOR STRETCH_BODY"
+    echo "export EDITOR='emacs -nw'" >> ~/.bashrc
```
Comment (hello-binit) [inline stretch_install_user.sh:13]: Users might not want Emacs. I switched to vim.

File: stretch_update.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```
Comment (hello-binit) [inline stretch_update.sh:1]: The first thing `stretch_update.sh` should do is reclone the stretch_install repository to ensure it's running the latest version of the script.

File: stretch_install_user.sh (hunk 1 above)
Comment (aedsinger) [reply stretch_install_user.sh:13]: This was carried forward from CK's script long ago. Do we even need it?

File: stretch_update.sh (hunk 2 above)
Comment (aedsinger) [reply stretch_update.sh:1]: Just added a git pull. That should be sufficient, right? Or a full clone is needed?

File: stretch_install_user.sh (hunk 1 above)
Comment (hello-binit) [reply stretch_install_user.sh:13]: No, we can get rid of it. Users can choose their preferred terminal editor.

File: stretch_update.sh (hunk 2 above)
Comment (hello-binit) [reply stretch_update.sh:1]: Actually, a git pull should work just fine.

File: stretch_install_user.sh (hunk 1 above)
Comment (aedsinger) [reply stretch_install_user.sh]: Removed it.

**Pull request merged**